frontend/src/metabase/timelines/actions.ts

```diff
@@ -0,0 +1,49 @@
+import { push } from "react-router-redux";
+import { createThunkAction } from "metabase/lib/redux";
+import * as Urls from "metabase/lib/urls";
+import Timelines from "metabase/entities/timelines";
+import TimelineEvents from "metabase/entities/timeline-events";
+import { Collection, Timeline, TimelineEvent } from "metabase-types/api";
+
+export const CREATE_TIMELINE = "metabase/timelines/CREATE_TIMELINE";
+export const createTimeline = createThunkAction(
+  CREATE_TIMELINE,
+  (values: Partial<Timeline>, collection: Collection) => {
```

Comment:
Wonder if we should add another `Timeline` type that only has fields required to create one
`Partial<Timeline>` would still accept an object the BE won't handle

Reply:
I think we probably should, but don't have good ideas how to do that. I'll leave it as it is for now - but come back in next PRs